src/empirical.jl

```diff
@@ -70,7 +70,7 @@ end
 
 ### fit model
 
-function fit_mle{T <: Real}(::Type{EmpiricalUnivariateDistribution},
-	                    x::Vector{T})
+function fit_mle(::Type{EmpiricalUnivariateDistribution},
+             x::Vector{T}) where T <: Real
```

Comment:
Keno pointed out that this is actually a mixed indent and FemtoCleaner assumes 4-space indents, so this change is expected.